includes/classes/DistributorPost.php

```diff
@@ -27,25 +27,6 @@
  * shown in IDEs.
  *
  * @since 2.0.0
- *
- * @method bool has_blocks()
- * @method bool has_block( string $block_name )
- * @method int  get_the_ID()
- * @method string get_permalink()
- * @method string get_post_type()
- * @method int|false get_post_thumbnail_id()
- * @method string|false get_post_thumbnail_url( string $size = 'post-thumbnail' )
- * @method string|false get_the_post_thumbnail( string $size = 'post-thumbnail', array $attr = '' )
- * @method string get_canonical_url( string $canonical_url = '' )
- * @method string get_author_name( string $author_name = '' )
- * @method string get_author_link( string $author_link = '' )
- * @method array get_meta()
- * @method array get_terms()
- * @method array get_media()
- * @method array post_data()
- * @method array to_insert( array $args = [] )
- * @method array to_pull_list( array $args = [] )
- * @method array to_rest( array $args = [] )
```

Comment:
This block is required to allow IDEs and docblock parsers to pick up the methods. They are all marked private so we can use a magic method to call them after ensuring the correct site is current on multisite.